controllers/hibp.js

```diff
@@ -13,6 +13,20 @@ const mozlog = require("../log");
 const log = mozlog("controllers.hibp");
 
 
+// Get address and language from either subscribers or
+// email_addresses fields
+function getAddressAndLanguageForEmail(recipient) {
+  let email = "";
```

Comment:
Not sure if we can do `let email = recipient.email;` and skip the `else` statement below.

Reply:
Yup, we can.